.env.development

```diff
@@ -36,6 +36,10 @@ LNWITH_URL=
 LOGIN_EMAIL_SERVER=smtp://mailhog:1025
 LOGIN_EMAIL_FROM=sndev@mailhog.dev
 
+# email salt
+# openssl rand -hex 32
+EMAIL_SALT=202c90943c313b829e65e3f29164fb5dd7ea3370d7262c4159691c2f6493bb8b
```

Comment:
This will need to be added to prod environments (with a different value, of course), however that works. But since it's a secret, I didn't add it to `.env.production`.